website/docs/d/application_security_group.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_application_security_group"
+sidebar_current: "docs-azurerm-datasource-network-application-security-group"
+description: |-
+  Get information about an Application Security Group.
+---
+
+# Data Source: azurerm_application_security_group
+
+Get information about an Application Security Group.
+
+-> **Note:** Application Security Groups are currently in Public Preview on an opt-in basis. [More information, including how you can register for the Preview, and which regions Application Security Groups are available in are available here](https://docs.microsoft.com/en-us/azure/virtual-network/create-network-security-group-preview)
+
+## Example Usage
+
+```hcl
+data "azurerm_application_security_group" "test" {
+  name = "tf-appsecuritygroup"
+  resource_group_name = "my-resource-group"
+}
+
+output "application_security_group_id" {
+  value = "${data.azurerm_application_security_group.test.id}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - The name of the Application Security Group.
+
```

Comment:
Does this extra whitespace in the bullets impact rendering?

It seems to on GitHub:

* test1

* test2

---

* test1
* test2

Reply:
not in our renderer, apparently - here's the source for Virtual Networks: https://raw.githubusercontent.com/terraform-providers/terraform-provider-azurerm/master/website/docs/r/virtual_network.html.markdown and the rendered version: https://www.terraform.io/docs/providers/azurerm/r/virtual_network.html